src/app/trajectory/map/map.page.ts

```diff
@@ -116,18 +118,31 @@ export class MapPage implements OnInit, OnDestroy {
 
     // TODO: rework this with optional inference type parameter,
     //   which we subscribe to and use to set zoom & open popup
-    if (history.state.center)
+    if (history.state.center) {
       this.mapBounds = latLng(history.state.center).toBounds(100)
+    }
   }
 
   onMapReady(map: Map) {
     this.map = map
   }
 
+  onMapMoved(map: Map) {
+    if (!this.isTogglingFollowPosition) {
```

Comment:
I don't understand why we need that extra boolean, mind to add a comment?

Reply:
I wanted to exit `followPosition`, when the map was moved by the user. `isTogglingFollowPosition` translates better to `suppressNextMapMoveEvent` (which I changed with the following commit).